docs-positions.ts

```diff
@@ -1,36 +1,37 @@
 export const docsPositions = [
   "docs/index.mdx",
+  "docs/installation.mdx",
```

Comment:
you can remove the changes to `docs-positions.ts`. I already updated this file and pushed up so its causing a merge conflict.